CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Master
+
+- Add `Search` component
+- 🚨 [Breaking change] Remove `txt-bold` and `txt-s` from `LevelIndicator` component to make it more flexible.
```

Comment:
When we update dr-ui in other repos using `LevelIndicator` we'll want to wrap the component in a span or div with these classes where needed.